test/GraphQL.Conventions.Tests/GraphQL.Conventions.Tests.csproj

```diff
@@ -25,6 +25,7 @@
     <PackageReference Include="NSubstitute" Version="4.3.0" />
     <PackageReference Include="ReportGenerator" Version="5.1.9" />
     <PackageReference Include="System.ComponentModel.TypeConverter" Version="4.3.0" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="7.*" />
```

Comment:
Includes `GetService<T>` and `GetRequiredService<T>`.  Including them within `GraphQL.Conventions` is problematic because if there is a `using` reference both namespaces, which will often happen in a `Startup.cs` file, then the compiler will not know which method to use.